ui/userMenu.js

```diff
@@ -0,0 +1,306 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
+/* exported UserMenu */
+/*
+ * Copyright © 2020 Endless OS Foundation LLC
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { AccountsService, Atk, Clutter, GLib, Gio, GObject, Pango, Shell, St } = imports.gi;
+
+const BoxPointer = imports.ui.boxpointer;
+const Main = imports.ui.main;
+const Panel = imports.ui.panel;
+const PanelMenu = imports.ui.panelMenu;
+const PopupMenu = imports.ui.popupMenu;
+const UserWidget = imports.ui.userWidget;
+
+const ExtensionUtils = imports.misc.extensionUtils;
+const PanelExtension = ExtensionUtils.getCurrentExtension();
+
+const USER_ICON_SIZE = 34;
+
+const SETTINGS_TEXT = _('Settings');
+const SETTINGS_LAUNCHER = 'gnome-control-center.desktop';
+
+const USER_ACCOUNTS_PANEL_LAUNCHER = 'gnome-user-accounts-panel.desktop';
+
+const ONLINE_ACCOUNTS_TEXT = _('Social Accounts');
+const ONLINE_ACCOUNTS_PANEL_LAUNCHER = 'gnome-online-accounts-panel.desktop';
+
+const FEEDBACK_TEXT = _('Give Us Feedback');
+const FEEDBACK_LAUNCHER = "eos-link-feedback.desktop";
+
+const HELP_CENTER_TEXT = _('Help');
+const HELP_CENTER_LAUNCHER = 'org.gnome.Yelp.desktop';
+const HELP_CENTER_LAUNCHER_ALT = 'yelp.desktop';
+
+const UserAccountSection = class extends PopupMenu.PopupMenuSection {
+    constructor(user) {
+        super();
+
+        // User account's icon
+        this.userIconItem = new PopupMenu.PopupBaseMenuItem({
```

Comment:
It is a good practice to keep field members private (thus, `this._userIconItem`) and add public getters and setters

Reply:
Done, just prefixed with "_" and updated the code using it (only in the same file so didn't bother adding getters/setters).